packages/devextreme/scss/widgets/fluent/form/_index.scss

```diff
@@ -70,10 +70,6 @@
   .dx-switch {
     vertical-align: middle;
   }
-
-  .dx-toggle-controls-paddings {
```

Comment:
I suggest you can remove '@use "../textEditor/sizes" as *;' import in this file. Check it please.

Reply:
If so, let's extract this change to separate PR.

Reply:
https://github.com/DevExpress/DevExtreme/pull/26075/commits/88f786367634684dae56d8868d421775fb90031d